Terminal.Gui/Views/TreeView.cs

```diff
@@ -0,0 +1,703 @@
+// This code is based on http://objectlistview.sourceforge.net (GPLv3 tree/list controls by phillip.piper@gmail.com)
```

Comment:
Please change this comment to explicitly note that Phillip has licensed this code under the MIT lic and not GPLv3.

Reply:
Have updated, let me know if you want me to adjust the phrasing.